backend/src/schema/resolvers/login.js

```diff
@@ -0,0 +1,46 @@
+const { AuthenticationError } = require("apollo-server-errors");
+
+const jwt = require("jsonwebtoken");
+
+const { userData } = require("../../data");
+
+module.exports.LoginResolver = {
+  Mutation: {
+    login: (object, params) => {
+      let user = userData.find(user => user.username === params.username);
+
+      if (user !== undefined) {
+        let token = jwt.sign(user, "supersecret");
```

Comment:
Bei unserer App ist Sicherheit jetzt nicht so relevant, aber es wäre sicherlich gut, das supersecret aus einer separaten file zu lesen (welche dann in Realität nicht commited wird)

Reply:
das stimmt natürlich. War jetzt aber nicht prio. Haben wir der Einfachheit halber weg gelassen